src/later.cpp

```diff
@@ -205,13 +205,9 @@ bool execCallbacksOne(
       break;
     }
 
-#ifdef RCPP_USING_UNWIND_PROTECT // See https://github.com/r-lib/later/issues/191
+    // See https://github.com/r-lib/later/issues/191
```

Comment:
Given no trace of or functions involving the unwind-protect, let's remove this comment (but keep the one above)


```suggestion
```